recipes/binarytree/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "binarytree" %}
+{% set version = "3.0.0" %}
+{% set sha256 = "3f46fa31db46146568241f55180e3d93e0b8f9bf5b8db5866f7208ca245f1acd" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/joowani/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  noarch: python
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+
+test:
+  imports:
+    - binarytree
```

Comment:
You might also try importing the exceptions

Reply:
Just imported the exceptions module as a whole.  Did you want to import them one by one instead?

Reply:
No that's fine, it's just a nice way to doubly check that the namespace installed correctly.